n2y/converter.py

```diff
@@ -163,18 +164,48 @@ def __init__(self, block):
         for key, value in block.items():
             self.__dict__[key] = value
 
+    def handler(self, annotation, target):
+        handler = {
+            'bold': [Strong([target]), Strong(target)],
+            'italic': [Emph([target]), Emph(target)],
+            'strikethrough': [Strikeout([target]), Strikeout(target)]
+        }
+        return handler[annotation]
+
+    def strip(self, target):
+        prependages = deque()
+        appendages = deque()
+        result = []
+        blank_space = [Space(), SoftBreak()]
+        nested_annotations = False
+        while target[0] in blank_space:
+            prependages.appendleft(target.pop(0))
```

Comment:
Logic error here. `prependages` ends up reversed here

Reply:
it's actually on purpose. I wanted to use the deque like a stack. I figure this way it left appends and now the first thing served in the for loop is the last thing added.

Reply:
Thanks, I was visualizing it incorrectly. The appendages _should_ be appended left though, right?

Reply:
You have two loops one going forward, and the other backward. When you reconstruct the result in the end, all of the pieces need to be arranged left to right. This is why on the reverse loop the items need to be added to the other side. In effect reversing a reversed list and making it correct.